src/status_im/browser/permissions.cljs

```diff
@@ -1,12 +1,12 @@
 (ns status-im.browser.permissions
   (:require [status-im.constants :as constants]
             [status-im.data-store.dapp-permissions :as dapp-permissions]
-            [status-im.ethereum.core :as ethereum]
             [status-im.extensions.module :as extensions.module]
             [status-im.i18n :as i18n]
             [status-im.qr-scanner.core :as qr-scanner]
             [status-im.ui.screens.navigation :as navigation]
-            [status-im.utils.fx :as fx]))
+            [status-im.utils.fx :as fx]
+            [status-im.ethereum.core :as ethereum]))
```

Comment:
why move it here it was previously alphabetically ordered

Reply:
let's ask cursive ide :)